packages/peregrine/lib/talons/Adapter/useAdapter.js

```diff
@@ -162,12 +162,12 @@ export const useAdapter = props => {
     const apolloLink = useMemo(
         () =>
             ApolloLink.from([
-                authLink,
-                errorLink,
-                httpLink,
                 mutationQueueLink,
                 retryLink,
-                storeLink
+                authLink,
+                storeLink,
+                errorLink,
+                httpLink
```

Comment:
The order of these is important to Apollo, with the terminating http link required to be at the end.

Reply:
Is this a bug?

Reply:
A bug introduced by the target branch; this change restores the link order that is present in mainline today.